src/vs/platform/terminal/node/terminalEnvironment.ts

```diff
@@ -72,3 +75,175 @@ export async function findExecutable(command: string, cwd?: string, paths?: stri
 	const fullPath = path.join(cwd, command);
 	return await exists(fullPath) ? fullPath : undefined;
 }
+
+export interface IShellIntegrationInjection {
```

Comment:
`IShellIntegrationLaunchConfig`?

I find injection confusing because this is before they've been injected, right?

Reply:
It defines what's going to be "injected", I think that's a little more clear as "launch config" has a pretty specific meaning in terminal land. I'll have a think about a more clear name